common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -746,6 +746,17 @@ class CelebornConf(loadDefaults: Boolean) extends Cloneable with Logging with Se
     get(WORKER_DIRECT_MEMORY_RATIO_FOR_SHUFFLE_STORAGE)
   def memoryPerResultPartition: String = get(MEMORY_PER_RESULT_PARTITION)
 
+  // //////////////////////////////////////////////////////
+  //                  Rate Limit controller              //
+  // //////////////////////////////////////////////////////
+  def workerRateLimitEnabled: Boolean = get(WORKER_RATE_LIMIT_ENABLED)
+  def workerRateLimitSampleTimeWindowSeconds: Long = get(WORKER_RATE_LIMIT_SAMPLE_TIME_WINDOW)
+  def workerRateLimitLowWatermark: Option[Long] = get(WORKER_RATE_LIMIT_LOW_WATERMARK)
```

Comment:
WORKER_RATE_LIMIT_LOW_WATERMARK and WORKER_RATE_LIMIT_HIGH_WATERMARK are related to WORKER_DIRECT_MEMORY_RATIO_PAUSE_RECEIVE/WORKER_DIRECT_MEMORY_RATIO_PAUSE_REPLICATE and WORKER_DIRECT_MEMORY_RATIO_RESUME, we'd better refine the logic among them. We can do it in later pr.